beetsplug/convert.py

```diff
@@ -89,12 +89,11 @@ def should_transcode(item, fmt):
     """Determine whether the item should be transcoded as part of
     conversion (i.e., its bitrate is high or it has the wrong format).
     """
-    no_convert_queries = config["convert"]["no_convert"].as_str_seq()
-    if no_convert_queries:
-        for query_string in no_convert_queries:
-            query, _ = parse_query_string(query_string, Item)
-            if query.match(item):
-                return False
+    no_convert_query = config["convert"]["no_convert"].as_str()
```

Comment:
I'd suggest creating a separate function for this check, something like:

```python
def in_no_convert(item: Item) -> bool:
    # check if item is excluded by the configured query
    ...

def should_transcode(item, fmt):
    """Determine whether the item should be transcoded as part of
    conversion (i.e., its bitrate is high or it has the wrong format).
    """
    if in_no_convert(item) or (
        config["convert"]["never_co...
```

This simplifies testing a lot, see the comment in the test file